pkg/registries/types/types.go

```diff
@@ -6,6 +6,38 @@ import (
 	"github.com/stackrox/rox/generated/storage"
 )
 
+const (
+	// ArtifactType represents the Google artifact image integration.
+	ArtifactType = "artifactregistry"
```

Comment:
nit: I think we should call this `ArtifactRegistry`. Artifact is a bit generic, and I feel easier to mistake for `Artifactory`